src/dmd/dsymbolsem.d

```diff
@@ -2743,44 +2743,44 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
          */
         if (f.isscope && !funcdecl.isNested() && !ad)
         {
-            funcdecl.error("functions cannot be scope");
+            funcdecl.error("functions cannot be `scope`");
         }
 
         if (f.isreturn && !funcdecl.needThis() && !funcdecl.isNested())
         {
             /* Non-static nested functions have a hidden 'this' pointer to which
              * the 'return' applies
              */
-            funcdecl.error("static member has no `this` to which `return` can apply");
+            funcdecl.error("`static` member has no `this` to which `return` can apply");
         }
 
         if (funcdecl.isAbstract() && !funcdecl.isVirtual())
         {
             const(char)* sfunc;
             if (funcdecl.isStatic())
-                sfunc = "static";
+                sfunc = "`static`";
             else if (funcdecl.protection.kind == Prot.Kind.private_ || funcdecl.protection.kind == Prot.Kind.package_)
                 sfunc = protectionToChars(funcdecl.protection.kind);
             else
                 sfunc = "non-virtual";
-            funcdecl.error("%s functions cannot be abstract", sfunc);
+            funcdecl.error("%s functions cannot be `abstract`", sfunc);
```

Comment:
Could `non-virtual` be inverted as `final`? So that you don't just highlight one of the cases here.

Reply:
I like it.  Done.